common/shlibs

```diff
@@ -3204,7 +3204,7 @@ libarcan_shmif_ext.so.0.11 arcan-0.5.4.3_1
 libarcan_shmif_server.so.0.11 arcan-0.5.4.3_1
 libarcan_tui.so.0.11 arcan-0.5.4.3_1
 liblwipv6.so.2 lwipv6-1.5a_1
-libpipewire-0.2.so.1 libpipewire-0.2.2_1
+libpipewire-0.3.so.0 libpipewire-0.3.4_1
```

Comment:
```suggestion
libpipewire-0.3.so.0 libpipewire-0.3.5_1
```

Reply:
also, shouldn't the other shared libraries (`libspa` etc) show up here?

Reply:
poked at this a bit more, all of the following `.so` files are currently included in the `pipewire` package and likely need to be added to `common/shlibs` and move to appropriate packages
```
/usr/lib/spa-0.2/alsa/libspa-alsa.so
/usr/lib/spa-0.2/audioconvert/libspa-audioconvert.so
/usr/lib/spa-0.2/audiomixer/libspa-audiomixer.so
/usr/lib/spa-0.2/control/libspa-control.so
/usr/lib/spa-0.2/jack/libspa-jack.so
/usr/lib/spa-0.2/support/libspa-dbus.so
/usr/lib/spa-0.2/support/libspa-support.so
/usr/lib/spa-0.2/v4l2/libspa-v4l2.so
/usr/lib/spa-0.2/videoconvert/libspa-videoconvert.so
/usr/lib/spa-0.2/vulkan/libspa-vulkan.so
```

Reply:
I don't think libspa are supposed to be linked (e.g. added to shlibs), at least for `alsa`, `jack`, `v4l2` and `vulkan`, as they're API plugins meant to be consumed by pipewire itself.

I considered moving alsa/jack/v4l2 and vulkan to a subpackage (just like `libspa-ffmpeg` and `libspa-bluez`) but v4l2 and alsa are actually included in `pipewire-0.2.7`. Not sure what would be the best way to do this, maybe with some defaults pipewire depends?